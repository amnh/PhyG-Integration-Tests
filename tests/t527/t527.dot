digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    24 -> 0 [label=0.0];
    24 -> 40 [label=0.0];
    25 -> 10 [label=56.0];
    25 -> 39 [label=2.0];
    26 -> 13 [label=9.0];
    26 -> 22 [label=9.0];
    27 -> 6 [label=50.0];
    27 -> 25 [label=5.0];
    28 -> 34 [label=47.0];
    28 -> 41 [label=29.0];
    29 -> 1 [label=1.0];
    29 -> 19 [label=8.0];
    30 -> 23 [label=4.0];
    30 -> 26 [label=14.0];
    31 -> 2 [label=68.0];
    31 -> 27 [label=20.0];
    32 -> 17 [label=50.0];
    32 -> 30 [label=17.0];
    33 -> 5 [label=31.0];
    33 -> 31 [label=1.0];
    34 -> 7 [label=13.0];
    34 -> 18 [label=5.0];
    35 -> 16 [label=58.0];
    35 -> 32 [label=2.0];
    36 -> 20 [label=38.0];
    36 -> 33 [label=23.0];
    37 -> 15 [label=33.0];
    37 -> 35 [label=1.0];
    38 -> 4 [label=18.0];
    38 -> 29 [label=1.0];
    39 -> 11 [label=50.0];
    39 -> 28 [label=4.0];
    40 -> 43 [label=0.0];
    40 -> 46 [label=0.0];
    41 -> 12 [label=3.0];
    41 -> 38 [label=5.0];
    42 -> 8 [label=0.0];
    42 -> 44 [label=24.0];
    43 -> 42 [label=0.0];
    43 -> 45 [label=0.0];
    44 -> 9 [label=4.0];
    44 -> 21 [label=36.0];
    45 -> 3 [label=22.0];
    45 -> 14 [label=23.0];
    46 -> 36 [label=0.0];
    46 -> 37 [label=3.0];
}
//1746.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    24 -> 0 [label=0.0];
    24 -> 39 [label=0.0];
    25 -> 10 [label=56.0];
    25 -> 38 [label=3.0];
    26 -> 13 [label=9.0];
    26 -> 22 [label=9.0];
    27 -> 6 [label=59.0];
    27 -> 25 [label=5.0];
    28 -> 33 [label=47.0];
    28 -> 41 [label=29.0];
    29 -> 1 [label=1.0];
    29 -> 19 [label=8.0];
    30 -> 26 [label=25.0];
    30 -> 40 [label=2.0];
    31 -> 17 [label=44.0];
    31 -> 30 [label=8.0];
    32 -> 5 [label=61.0];
    32 -> 27 [label=0.0];
    33 -> 7 [label=13.0];
    33 -> 18 [label=5.0];
    34 -> 16 [label=49.0];
    34 -> 31 [label=7.0];
    35 -> 20 [label=37.0];
    35 -> 32 [label=4.0];
    36 -> 15 [label=27.0];
    36 -> 34 [label=5.0];
    37 -> 4 [label=18.0];
    37 -> 29 [label=1.0];
    38 -> 11 [label=51.0];
    38 -> 28 [label=7.0];
    39 -> 45 [label=1.0];
    39 -> 46 [label=10.0];
    40 -> 14 [label=0.0];
    40 -> 23 [label=0.0];
    41 -> 12 [label=3.0];
    41 -> 37 [label=5.0];
    42 -> 8 [label=0.0];
    42 -> 44 [label=24.0];
    43 -> 3 [label=0.0];
    43 -> 42 [label=0.0];
    44 -> 9 [label=4.0];
    44 -> 21 [label=36.0];
    45 -> 2 [label=65.0];
    45 -> 43 [label=3.0];
    46 -> 35 [label=0.0];
    46 -> 36 [label=0.0];
}
//1729.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    24 -> 0 [label=0.0];
    24 -> 40 [label=0.0];
    25 -> 9 [label=4.0];
    25 -> 22 [label=5.0];
    26 -> 13 [label=9.0];
    26 -> 25 [label=1.0];
    27 -> 6 [label=78.0];
    27 -> 39 [label=0.0];
    28 -> 34 [label=47.0];
    28 -> 41 [label=29.0];
    29 -> 1 [label=2.0];
    29 -> 19 [label=8.0];
    30 -> 23 [label=4.0];
    30 -> 26 [label=14.0];
    31 -> 2 [label=71.0];
    31 -> 27 [label=26.0];
    32 -> 17 [label=56.0];
    32 -> 30 [label=17.0];
    33 -> 5 [label=40.0];
    33 -> 31 [label=1.0];
    34 -> 7 [label=13.0];
    34 -> 18 [label=5.0];
    35 -> 16 [label=57.0];
    35 -> 32 [label=2.0];
    36 -> 20 [label=34.0];
    36 -> 33 [label=27.0];
    37 -> 15 [label=31.0];
    37 -> 35 [label=1.0];
    38 -> 4 [label=19.0];
    38 -> 29 [label=1.0];
    39 -> 11 [label=34.0];
    39 -> 28 [label=18.0];
    40 -> 43 [label=0.0];
    40 -> 46 [label=0.0];
    41 -> 12 [label=3.0];
    41 -> 38 [label=5.0];
    42 -> 8 [label=0.0];
    42 -> 21 [label=63.0];
    43 -> 42 [label=0.0];
    43 -> 44 [label=0.0];
    44 -> 14 [label=21.0];
    44 -> 45 [label=9.0];
    45 -> 3 [label=11.0];
    45 -> 10 [label=66.0];
    46 -> 36 [label=0.0];
    46 -> 37 [label=2.0];
}
//1716.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    24 -> 0 [label=0.0];
    24 -> 40 [label=0.0];
    25 -> 9 [label=4.0];
    25 -> 22 [label=5.0];
    26 -> 13 [label=9.0];
    26 -> 25 [label=1.0];
    27 -> 6 [label=78.0];
    27 -> 39 [label=0.0];
    28 -> 34 [label=47.0];
    28 -> 42 [label=29.0];
    29 -> 1 [label=2.0];
    29 -> 19 [label=8.0];
    30 -> 26 [label=25.0];
    30 -> 41 [label=2.0];
    31 -> 2 [label=72.0];
    31 -> 27 [label=26.0];
    32 -> 17 [label=37.0];
    32 -> 30 [label=9.0];
    33 -> 5 [label=39.0];
    33 -> 31 [label=1.0];
    34 -> 7 [label=13.0];
    34 -> 18 [label=5.0];
    35 -> 16 [label=50.0];
    35 -> 32 [label=9.0];
    36 -> 20 [label=37.0];
    36 -> 33 [label=25.0];
    37 -> 15 [label=30.0];
    37 -> 35 [label=4.0];
    38 -> 4 [label=19.0];
    38 -> 29 [label=1.0];
    39 -> 11 [label=34.0];
    39 -> 28 [label=17.0];
    40 -> 44 [label=3.0];
    40 -> 46 [label=1.0];
    41 -> 14 [label=0.0];
    41 -> 23 [label=0.0];
    42 -> 12 [label=3.0];
    42 -> 38 [label=5.0];
    43 -> 8 [label=0.0];
    43 -> 21 [label=63.0];
    44 -> 43 [label=0.0];
    44 -> 45 [label=0.0];
    45 -> 3 [label=7.0];
    45 -> 10 [label=72.0];
    46 -> 36 [label=0.0];
    46 -> 37 [label=5.0];
}
//1714.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    47 [label=HTU47];
    48 [label=HTU48];
    24 -> 0 [label=0.0];
    24 -> 38 [label=0.0];
    25 -> 13 [label=9.0];
    25 -> 22 [label=10.0];
    26 -> 6 [label=80.0];
    26 -> 37 [label=0.0];
    27 -> 32 [label=47.0];
    27 -> 40 [label=29.0];
    28 -> 1 [label=2.0];
    28 -> 19 [label=8.0];
    29 -> 25 [label=25.0];
    29 -> 39 [label=0.0];
    30 -> 17 [label=42.0];
    30 -> 29 [label=9.0];
    31 -> 5 [label=68.0];
    31 -> 26 [label=4.0];
    32 -> 7 [label=13.0];
    32 -> 18 [label=5.0];
    33 -> 16 [label=49.0];
    33 -> 30 [label=9.0];
    34 -> 20 [label=37.0];
    34 -> 31 [label=5.0];
    35 -> 15 [label=29.0];
    35 -> 33 [label=6.0];
    36 -> 4 [label=19.0];
    36 -> 28 [label=1.0];
    37 -> 11 [label=33.0];
    37 -> 27 [label=19.0];
    38 -> 46 [label=1.0];
    38 -> 48 [label=5.0];
    39 -> 23 [label=4.0];
    39 -> 47 [label=0.0];
    40 -> 12 [label=3.0];
    40 -> 36 [label=5.0];
    41 -> 8 [label=0.0];
    41 -> 43 [label=24.0];
    42 -> 41 [label=0.0];
    42 -> 44 [label=0.0];
    43 -> 9 [label=4.0];
    43 -> 21 [label=36.0];
    44 -> 45 [label=0.0];
    44 -> 47 [label=0.0];
    45 -> 3 [label=4.0];
    45 -> 10 [label=78.0];
    46 -> 2 [label=65.0];
    46 -> 42 [label=3.0];
    47 -> 14 [label=0.0];
    48 -> 34 [label=0.0];
    48 -> 35 [label=3.0];
}
//1711.0
digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=Antarctic_neomeniomorph];
    1 [label=Aplysia_californica];
    2 [label=Cerebratulus_lacteus];
    3 [label=Chaetopleura_apiculata];
    4 [label=Crassostrea_virginica];
    5 [label=Euprymna_scolopes];
    6 [label=Greenland_neomeniomorph];
    7 [label=Helobdella_robusta];
    8 [label=Ilyanassa_obsoleta];
    9 [label=Lingula_anatina];
    10 [label=Littorina_littorea];
    11 [label=Lottia_gigantea];
    12 [label=Lymnaea_stagnalis];
    13 [label=Mytilus_californianus];
    14 [label=Nautilus];
    15 [label=Nucula];
    16 [label=Octopus];
    17 [label=Perotrochus_lucaya];
    18 [label=Scaphopods];
    19 [label=Siphonaria_pectinata];
    20 [label=Solemya];
    21 [label=Venerupis_decussatus];
    22 [label=Venerupis_philippinarum];
    23 [label=Yoldia_limatula];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    37 [label=HTU37];
    38 [label=HTU38];
    39 [label=HTU39];
    40 [label=HTU40];
    41 [label=HTU41];
    42 [label=HTU42];
    43 [label=HTU43];
    44 [label=HTU44];
    45 [label=HTU45];
    46 [label=HTU46];
    24 -> 0 [label=0.0];
    24 -> 38 [label=0.0];
    25 -> 13 [label=9.0];
    25 -> 22 [label=10.0];
    26 -> 6 [label=80.0];
    26 -> 37 [label=0.0];
    27 -> 32 [label=47.0];
    27 -> 40 [label=29.0];
    28 -> 1 [label=2.0];
    28 -> 19 [label=8.0];
    29 -> 25 [label=25.0];
    29 -> 39 [label=2.0];
    30 -> 17 [label=42.0];
    30 -> 29 [label=9.0];
    31 -> 5 [label=68.0];
    31 -> 26 [label=4.0];
    32 -> 7 [label=13.0];
    32 -> 18 [label=5.0];
    33 -> 16 [label=49.0];
    33 -> 30 [label=9.0];
    34 -> 20 [label=37.0];
    34 -> 31 [label=5.0];
    35 -> 15 [label=29.0];
    35 -> 33 [label=6.0];
    36 -> 4 [label=19.0];
    36 -> 28 [label=1.0];
    37 -> 11 [label=33.0];
    37 -> 27 [label=19.0];
    38 -> 45 [label=1.0];
    38 -> 46 [label=5.0];
    39 -> 14 [label=0.0];
    39 -> 23 [label=0.0];
    40 -> 12 [label=3.0];
    40 -> 36 [label=5.0];
    41 -> 8 [label=0.0];
    41 -> 43 [label=24.0];
    42 -> 41 [label=0.0];
    42 -> 44 [label=0.0];
    43 -> 9 [label=4.0];
    43 -> 21 [label=36.0];
    44 -> 3 [label=4.0];
    44 -> 10 [label=78.0];
    45 -> 2 [label=65.0];
    45 -> 42 [label=3.0];
    46 -> 34 [label=0.0];
    46 -> 35 [label=3.0];
}
//1711.0
