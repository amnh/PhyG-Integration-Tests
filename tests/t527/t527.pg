--build(distance, block, graph), inputting prealigned amino acid file
set(seed:1634561640)
set(graphtype:softwired)
read(preaminoacid:"metazoa-aa-1.fas")
read(preaminoacid:"metazoa-aa-2.fas")
build(character, replicates:10, block, cun, displaytrees:5, atrandom, graph)
report("t527.csv", data, overwrite)
report("t527.tre", newick, graphs, overwrite)
report("t527.dot", dotpdf, overwrite)