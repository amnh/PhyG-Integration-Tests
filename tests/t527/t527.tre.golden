(Antarctic_neomeniomorph:120.5,(Perotrochus_lucaya:44.0,(Nautilus:93.0,(Lottia_gigantea:86.5,((Lingula_anatina:207.0,(Venerupis_philippinarum:208.5,((Scaphopods:189.5,(Ilyanassa_obsoleta:193.5,(Crassostrea_virginica:228.5,(Lymnaea_stagnalis:194.0,(Aplysia_californica:80.0,Helobdella_robusta:207.0)HTU29:6.5)HTU28:11.0)HTU27:8.0)HTU26:50.0)HTU25:63.5,(Greenland_neomeniomorph:141.5,(Solemya:102.0,(Cerebratulus_lacteus:183.5,Mytilus_californianus:64.5)HTU32:20.5)HTU30:31.0)HTU31:50.5)HTU33:13.5)HTU34:9.0)HTU35:35.5,((Octopus:147.5,(Chaetopleura_apiculata:87.5,Siphonaria_pectinata:213.5)HTU41:22.0)HTU38:36.0,(Littorina_littorea:110.5,((Euprymna_scolopes:106.0,Yoldia_limatula:202.5)HTU37:55.0,(Nucula:100.5,Venerupis_decussatus:212.0)HTU43:74.0)HTU39:29.0)HTU42:11.0)HTU40:12.5)HTU36:15.0)HTU44:14.0)HTU45:24.0)HTU46:56.5)HTU24[1786.0];
(Antarctic_neomeniomorph:120.5,(Perotrochus_lucaya:43.5,(Nautilus:94.5,(Lottia_gigantea:90.0,((Lingula_anatina:207.5,(Venerupis_philippinarum:209.0,((Scaphopods:189.5,(Ilyanassa_obsoleta:193.5,(Crassostrea_virginica:228.5,(Lymnaea_stagnalis:194.0,(Aplysia_californica:79.5,Helobdella_robusta:207.0)HTU29:6.0)HTU28:10.5)HTU27:7.5)HTU26:50.0)HTU25:62.0,(Greenland_neomeniomorph:147.0,(Cerebratulus_lacteus:180.5,Mytilus_californianus:68.5)HTU31:30.0)HTU30:44.5)HTU32:12.0)HTU33:8.5)HTU34:33.5,((Octopus:147.5,(Chaetopleura_apiculata:87.0,Siphonaria_pectinata:213.5)HTU40:22.5)HTU37:36.0,(Littorina_littorea:112.0,((Euprymna_scolopes:104.5,Yoldia_limatula:202.5)HTU36:54.0,(Solemya:94.5,(Nucula:85.0,Venerupis_decussatus:212.0)HTU43:17.0)HTU42:92.0)HTU38:29.0)HTU41:11.5)HTU39:14.0)HTU35:15.5)HTU44:12.5)HTU45:21.5)HTU46:57.0)HTU24[1778.0];
(Antarctic_neomeniomorph:119.5,(Perotrochus_lucaya:47.5,(Nautilus:102.0,(Lottia_gigantea:85.5,((Lingula_anatina:208.0,(Venerupis_philippinarum:209.5,((Octopus:137.5,(Euprymna_scolopes:134.0,(Scaphopods:191.0,(Ilyanassa_obsoleta:192.0,(Crassostrea_virginica:227.0,(Lymnaea_stagnalis:192.5,(Aplysia_californica:81.0,Helobdella_robusta:205.5)HTU29:9.5)HTU28:14.0)HTU27:11.0)HTU26:50.5)HTU25:48.5)HTU30:28.0)HTU31:49.0,(Cerebratulus_lacteus:175.5,Mytilus_californianus:70.5)HTU32:37.0)HTU33:21.0)HTU34:17.5)HTU35:41.5,((Chaetopleura_apiculata:93.0,Siphonaria_pectinata:214.5)HTU40:37.5,(Littorina_littorea:95.5,((Greenland_neomeniomorph:113.0,Yoldia_limatula:202.0)HTU37:48.0,(Solemya:92.5,(Nucula:85.5,Venerupis_decussatus:211.5)HTU43:18.0)HTU42:75.5)HTU38:52.5)HTU41:31.0)HTU39:27.0)HTU36:18.0)HTU44:16.0)HTU45:25.0)HTU46:59.0)HTU24[1757.0];
(Antarctic_neomeniomorph:119.5,(Perotrochus_lucaya:55.0,(Nautilus:103.0,(Lottia_gigantea:66.5,((Lingula_anatina:208.5,(Venerupis_philippinarum:210.0,((Octopus:129.5,(Euprymna_scolopes:132.0,(Scaphopods:205.5,(Ilyanassa_obsoleta:196.5,(Crassostrea_virginica:231.5,(Lymnaea_stagnalis:197.0,(Aplysia_californica:120.5,(Chaetopleura_apiculata:78.5,Helobdella_robusta:201.5)HTU30:18.0)HTU29:15.0)HTU28:14.0)HTU27:11.0)HTU26:20.0)HTU25:37.0)HTU31:30.5)HTU32:47.0,(Cerebratulus_lacteus:176.5,Mytilus_californianus:69.0)HTU33:33.5)HTU34:19.0)HTU35:14.5)HTU36:42.5,(Siphonaria_pectinata:232.5,(Littorina_littorea:118.5,((Greenland_neomeniomorph:111.0,Yoldia_limatula:202.5)HTU38:49.0,(Solemya:91.5,(Nucula:85.5,Venerupis_decussatus:212.0)HTU43:18.5)HTU42:73.5)HTU39:37.0)HTU41:18.5)HTU40:22.0)HTU37:20.5)HTU44:25.5)HTU45:36.5)HTU46:58.0)HTU24[1727.0];
(Antarctic_neomeniomorph:119.5,(Perotrochus_lucaya:51.5,(Nautilus:101.0,(Lottia_gigantea:71.0,((Lingula_anatina:208.5,(Venerupis_philippinarum:210.0,(((Scaphopods:208.5,(Ilyanassa_obsoleta:196.5,(Crassostrea_virginica:231.5,(Lymnaea_stagnalis:197.0,(Aplysia_californica:120.0,(Helobdella_robusta:211.5,(Chaetopleura_apiculata:4.0)#HTU46:4.0)HTU30:7.5)HTU29:13.0)HTU28:11.5)HTU27:8.5)HTU26:18.5)HTU25:48.5,(Euprymna_scolopes:6.0)#HTU47:6.0)HTU31:26.0,(Cerebratulus_lacteus:171.0,Mytilus_californianus:74.5)HTU32:38.0)HTU33:15.5)HTU34:11.0)HTU35:40.5,((Octopus:104.5,(Siphonaria_pectinata:213.0,#HTU46:1.5)HTU42:20.0)HTU39:56.5,(Littorina_littorea:115.0,((Greenland_neomeniomorph:111.5,(Yoldia_limatula:201.5,#HTU47:1.5)HTU38:23.0)HTU37:48.0,(Solemya:91.0,(Nucula:85.5,Venerupis_decussatus:211.5)HTU45:19.0)HTU44:75.0)HTU40:38.5)HTU43:23.0)HTU41:26.0)HTU36:16.5)HTU48:20.0)HTU49:31.5)HTU50:58.0)HTU24[1725.0];
(Antarctic_neomeniomorph:119.5,(Perotrochus_lucaya:51.5,(Nautilus:101.0,(Lottia_gigantea:71.0,((Lingula_anatina:208.5,(Venerupis_philippinarum:210.0,((Euprymna_scolopes:138.5,(Scaphopods:208.5,(Ilyanassa_obsoleta:196.5,(Crassostrea_virginica:231.5,(Lymnaea_stagnalis:197.0,(Aplysia_californica:120.0,(Chaetopleura_apiculata:79.0,Helobdella_robusta:201.5)HTU30:17.5)HTU29:13.0)HTU28:11.5)HTU27:8.5)HTU26:18.5)HTU25:32.0)HTU31:42.5,(Cerebratulus_lacteus:171.0,Mytilus_californianus:74.5)HTU32:38.0)HTU33:15.5)HTU34:11.0)HTU35:40.5,((Octopus:104.5,Siphonaria_pectinata:213.0)HTU38:56.5,(Littorina_littorea:115.0,((Greenland_neomeniomorph:111.5,Yoldia_limatula:201.5)HTU37:48.0,(Solemya:91.0,(Nucula:85.5,Venerupis_decussatus:211.5)HTU43:19.0)HTU42:75.0)HTU39:38.5)HTU41:23.0)HTU40:26.0)HTU36:16.5)HTU44:20.0)HTU45:31.5)HTU46:58.0)HTU24[1725.0];
